src/c++/library/http_client.cc

```diff
@@ -26,17 +26,19 @@
 
 // Include this first to make sure we are a friend of common classes.
 #define TRITON_INFERENCE_SERVER_CLIENT_CLASS InferenceServerHttpClient
-#include "common.h"
+#include "http_client.h"
 
 #include <curl/curl.h>
+
```

Comment:
Remove blank space.

Reply:
Auto-formatter. Do you want me to save without formatting?

Reply:
There is no standard currently for what formatter to use. And we have no mechanism currently for codifying proposed standards.

Reply:
There is a standard for code format: https://github.com/triton-inference-server/server/blob/5d2bcf84c83f241b01cda73457e8415fabc4f5f6/CONTRIBUTING.md#coding-convention

Are you using clang-format-6.0?

Reply:
As another datapoint, I explicitly removed those lines and saved the file, and it did NOT add the blank lines back in. 

I also tried to reorder all of the includes, and it DID reorder them back into the original sorting, so my format-on-save is working.

Reply:
Undid auto-formatting changes 👍